cgmanifests/generate_cgmanifest.py

```diff
@@ -59,6 +59,7 @@ def add_github_dep(name, parsed_url):
         if dep not in git_deps:
             git_deps[dep] = name
     else:
+        # TODO: support urls like: https://github.com/onnx/onnx-tensorrt/archive/refs/tags/release/7.1.zip
```

Comment:
this is supported now after merging in snnn's fix , right?

Reply:
But it is not needed for this change.

Reply:
Right now this script supports URLs like:
```
https://github.com/onnx/onnx-tensorrt/archive/refs/tags/***.zip
```
But not 
```
https://github.com/onnx/onnx-tensorrt/archive/refs/tags/***/***.zip
```

Reply:
ok. thanks for the clarification.